osimpipeline/study.py

```diff
@@ -17,25 +17,67 @@ def __init__(self, trial, num, gait_landmarks, metadata=None):
         self.num = num
         self.name = 'cycle%02i' % num
         self.metadata = metadata
-        self.rel_path = os.path.join(trial.rel_path, self.name)
         self.id = '_'.join([trial.id, self.name])
```

Comment:
I think it's right to remove `rel_path` now; should `self.id` be removed as well?